styleguide/derek/incubation/_layout.jade

```diff
@@ -1,4 +1,11 @@
 extends ../../_layouts/page
 
 block content
-  != yield
+  !=partial("../_partials/_ceiling")
+  !=partial("../_partials/_mainnav")
+  .contentOffset
+    != yield
+  !=partial("../_partials/_back-to-top")
+  !=partial("../_partials/_rug")
+  !=partial("../_partials/_footer")
+  !=partial("../global-components/basement/basement")
```

Comment:
These paths are all changing. Several of these changes have already been merged in, so rebase and confirm on your end.